src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import PersonItem from '../components/PersonItem';
+
+const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getPeople()
+      .then(result => {
+        const personWithParents = result.map(person => {
+          const modifiedPerson = {...person};
+          const mam = result.find(p => p.name === person.motherName);
+          const dad = result.find(p => p.name === person.fatherName);
+
+          if (mam) {
+            modifiedPerson.mother = mam;
+          }
+
+          if (dad) {
+            modifiedPerson.father = dad;
+          }
+
+          return modifiedPerson;
+        });
+
+        setPeople(personWithParents);
+      })
+      .catch(() => {
+        setIsError(true);
+        setTimeout(() => {
+          setIsError(false);
+        }, 3000);
```

Comment:
The error handling logic sets `isError` to `false` after 3 seconds, which might not be ideal if the user doesn't notice the error message in time. Consider keeping the error message visible until the user takes an action, like retrying the fetch, or provide a way for the user to dismiss the error message manually.